src/Uno.UI/UI/Xaml/Window.wasm.cs

```diff
@@ -166,7 +166,7 @@ private void InternalSetContent(UIElement content)
 
 				if (FeatureConfiguration.FrameworkElement.WasmUseManagedLoadedUnloaded && !_window.IsLoaded)
 				{
-					_window.ManagedOnLoaded();
+					_window.ManagedOnLoaded(1);
```

Comment:
On wasm there's a new depth counter for logging/tracing purposes. On other platforms it would be costly to add, but on Wasm it's pretty easy because were're controlling the whole hierarchy.